[DOCS-6337] CCM Updating Azure Setup Doc (#20002)
DataDog/documentation: content/en/cloud_cost_management/_index.md

Datadog ingests your cloud cost data and transforms it into queryable metrics. If costs rise, you can correlate the change with usage metrics to determine the root cause.

## Setup

{{< whatsnext desc="Get started with Cloud Cost Management:">}}
  {{< nextlink href="/cloud_cost_management/aws">}}<u>AWS</u>: Configure Cloud Cost Management for your AWS bill.{{< /nextlink >}}
  {{< nextlink href="/cloud_cost_management/azure">}}<u>Azure</u>: Configure Cloud Cost Management for your Azure bill. {{< /nextlink >}}
 {{< /whatsnext >}}

## Cloud costs in dashboards

> Visualizing infrastructure spend alongside related utilization metrics can help you spot potential inefficiencies and savings opportunities. You can add cloud costs to widgets in Datadog dashboards by selecting the *Cloud Cost* data source.

Reviewer: ```suggestion
Visualize infrastructure spend alongside related utilization metrics to spot potential inefficiencies and savings opportunities:
- Add cloud costs to widgets in Datadog dashboards by selecting the *Cloud Cost* data source. 
- Export timeseries costs data using the [Metrics API][1].

```

{{< img src="cloud_cost/cloud_cost_data_source.png" alt="Cloud Cost available as a data source in dashboard widget creation" >}}

> Customers can export timeseries costs data using the [Metrics API][1]

Reviewer: ```suggestion
```

> [1]: https://docs.datadoghq.com/api/latest/metrics/#query-timeseries-data-across-multiple-products

Reviewer: Move this below the `Further Reading` section
```suggestion
```

## Further reading

> {{< partial name="whats-next/whats-next.html" >}}

Reviewer: ```suggestion
{{< partial name="whats-next/whats-next.html" >}}

[1]: https://docs.datadoghq.com/api/latest/metrics/#query-timeseries-data-across-multiple-products

```
